core/src/navigation/LogoTitle.html

```diff
@@ -84,7 +116,7 @@
             role="button"
             class="fd-menu__link"
             on:click|preventDefault="{() => goTo(item.link)}"
-            href="{getRouteLink(item, pathParams)}"
+            href="{addNavHrefForAnchor ? getRouteLink(item, pathParams) : undefined}"
```

Comment:
I commented on a very similar case [here](https://github.com/SAP/luigi/pull/2172/files#r683521273) that **addNavHrefForAnchor** could be redundant. Since I'm seeing it here as well, maybe that's due to some reason I am not aware of?